clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -207,6 +221,18 @@ func (p *podStartupLatencyMeasurement) gather(c clientset.Interface) ([]measurem
 	var err error
 	if successRatio := float32(len(startupLag)) / float32(len(p.createTimes)); successRatio < successfulStartupRatioTreshold {
 		err = fmt.Errorf("only %v%% of all pods were scheduled successfully", successRatio*100)
+		glog.Errorf(err.Error())
```

Comment:
I think glog.Errorf(err) is what you want

Reply:
Fixed.